source/guides/administrator.rst

```diff
@@ -205,3 +205,15 @@ Documentation on early previews and unofficial functionality.
    :glob:
 
    /install/prod-windows-2012*
+   
+   Mattermost Archive
+----------------------------
+
+Archived documents.
```

Comment:
Maybe a short description that this includes archived changelogs and versions.

Reply:
I add something but wait for the push